WaterNetworkManagement/Network/doc/spec.md

```diff
@@ -38,6 +38,13 @@ A JSON Schema corresponding to this data model can be found [here](../schema.jso
         -   `{{metadata Property name}}` : {{Metadata Property Description}}
     -   Mandatory
 
+-   `hasSubGraph` : The ID of the sub-network, considered as a Network.
```

Comment:
Just wondering, why is this `hasSubGraph` rather than a `hasSubNetwork`? The latter seems more logical?

Reply:
yes. Updated.